satpy/Combined_colormaps.ipynb

```diff
@@ -0,0 +1,541 @@
+{
```

Comment:
<p>I'd like to avoid referring to issues if we can. It sounds like this is something that should be explained better in the satpy docs. Maybe we need to make a PR there first to clear this up.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>